tests/test-spawn.lua

```diff
@@ -10,6 +10,31 @@ local exit_yay, exit_snd = nil, nil
 -- * spawn with startup notification is covered by test-spawn-snid.lua
 
 local steps = {
+  function()
+    -- Test various error conditions. There are quite a number of them...
+    local error_message
+
+    error_message = spawn("this_does_not_exist_and_should_fail")
+    assert(string.find(error_message, 'No such file or directory'), error_message)
+
+    error_message = spawn{"this_does_not_exist_and_should_fail"}
+    assert(string.find(error_message, 'No such file or directory'), error_message)
```

Comment:
One is with a string as argument, the other with a table. They take different code paths in the C code (one of them wants to turn the string into a "list of strings", the other directly takes the table to get that "list of strings").

Any idea how to make this difference more prominent?